### PR TITLE
835 Ubuntu 18.04 Support

### PR DESCRIPTION
**Pull request by ryanRfox**
Update README.md associated with #835

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,15 +41,26 @@ To build after all dependencies are installed:
...
+**Upgrade Script** (prepend to the Build Script above if you built a prior release):
 
-**NOTE:** BitShares requires a [Boost](http://www.boost.org/) version in the range [1.57, 1.65]. Versions earlier than
-1.57 or newer than 1.65 are NOT supported. If your system Boost version is newer, then you will need to manually build
+    git remote set-url origin https://github.com/bitshares/bitshares-core.git
+    git checkout master
+    git remote set-head origin --auto
+    git pull
+    git submodule sync --recursive
+    git submodule update --init --recursive
```
Comment (abitmore) [inline README.md]: Perhaps need one more `git submodule update --init --recursive` before `sync`. More info: https://github.com/bitshares/bitshares-core/releases/tag/2.0.180328

File: README.md [hunk 2]
```diff
@@ -49,15 +41,26 @@ To build after all dependencies are installed:
...
+
+**NOTE:** BitShares requires a [Boost](http://www.boost.org/) version in the range [1.57 - 1.65]. Versions earlier than
+1.57 or newer than 1.65 are NOT supported. If your system's Boost version is newer, then you will need to manually build
```
Comment (pmconrad) [inline README.md]: Actually we support 1.65.1

File: README.md [hunk 3]
```diff
@@ -49,15 +41,27 @@ To build after all dependencies are installed:
     cmake -DCMAKE_BUILD_TYPE=RelWithDebInfo .
```
Comment (abitmore) [inline README.md]: `<LATEST_RELEASE_TAG>` has confused some people. Perhaps update it after every release? Or remove that line, then by default `master` branch will be used (but `--version` command won't show the correct tag).

File: README.md [hunk 4]
```diff
@@ -49,15 +41,27 @@ To build after all dependencies are installed:
...
-**NOTE:** BitShares requires an [OpenSSL](https://www.openssl.org/) version in the 1.0.x series. OpenSSL 1.1.0 and newer are NOT supported. If your system OpenSSL version is newer, then you will need to manually provide an older version of OpenSSL and specify it to CMake using `-DOPENSSL_INCLUDE_DIR`, `-DOPENSSL_SSL_LIBRARY`, and `-DOPENSSL_CRYPTO_LIBRARY`.
+**Upgrade Script** (prepend to the Build Script above if you built a prior release):
 
-**NOTE:** BitShares requires a [Boost](http://www.boost.org/) version in the range [1.57, 1.65]. Versions earlier than
-1.57 or newer than 1.65 are NOT supported. If your system Boost version is newer, then you will need to manually build
+    git remote set-url origin https://github.com/bitshares/bitshares-core.git
+    git checkout master
+    git remote set-head origin --auto
+    git pull
+    git submodule update --init --recursive # this command will fail
```
Comment (abitmore) [inline README.md]: `this command will fail` is not always true.

Comment (ryanRfox) [reply README.md]: change it to to "this command _may_ fail" will be best, else eliminate the comment?

File: README.md (hunk 3 above)
Comment (ryanRfox) [reply README.md]: Agreed, let's make it simple for the end user to copy paste the script. I've added a new `current_version` file containing the text matching the (expected) release tag. I've swapped the <variable> for a $variable and inserted a bash script to set `variable=$(curl URL)`

**Comment by ryanRfox** [comment]
If we accept this pull with the $LATEST_RELEASE_TAG included, we must update our Git Flow document to include this step with each release.

File: README.md (hunk 3 above)
Comment (ryanRfox) [reply README.md]: Removed the `curl` statement and `$variable`. Replaced with an explicit `git checkout master` and added a comment about replacing with current release as needed.

**Pull request merged**